code/game/world.dm

```diff
@@ -22,14 +22,16 @@ GLOBAL_LIST_INIT(map_transition_config, MAP_TRANSITION_CONFIG)
 
 	TgsNew(new /datum/tgs_event_handler/impl, TGS_SECURITY_TRUSTED) // creates a new TGS object
 	log_world("World loaded at [time_stamp()]")
-	log_world("[GLOB.vars.len - GLOB.gvars_datum_in_built_vars.len] global variables")
+	log_world("[length(GLOB.vars) - length(GLOB.gvars_datum_in_built_vars)] global variables")
 	GLOB.revision_info.log_info()
-	load_admins(run_async=FALSE) // This better happen early on.
+	load_admins(FALSE) // This better happen early on.
```

Comment:
Again, nitpicky but please keep the actual words "run_aysnc" just to make it absolutely abundantly clear. These parts of code aren't modified often, and should be very obvious for where mistakes can be made.

Reply:
Don't mind nitpicks, especially since this is a pretty important bit of the code.